### PR TITLE
docs(concepts): Describe audit trail feature

### LINKED ISSUE #3739: Improve docs for audit trail
ZITADEL docs currently do not explain our audit log. 

We have only some docs about [Eventstore](https://docs.zitadel.com/docs/concepts/eventstore) and [Software Architecture](https://docs.zitadel.com/docs/concepts/architecture#software-architecture)

From a business perspective we should maybe answer the following questions: 
- How do we ensure auditability of changes to IAM objects (attribute-level)?
- How to access information in the audit trail via GUI?
- How to access audit information via APIs (incl. ~planned~ features like event-API)?

We could include the following as additional info: 
- How eventstore provides complete and clean data
- Future plans for standard reports
- ~Future plans for~ APIs (and why pull vs. push to handle backpressure of an HA system)
- Using / sending data to external log system
- In-context: Allows to view the overall system at one specific point in time

### PR DESCRIPTION
**Pull request by mffap**
closes #3739 

dependent on #5800

### CODE REVIEW

**Review by hifabienne — COMMENTED** [review]
Looks good to me. You links some images that are not in the pr.
Not sure if they already exist or if they are missing.

**Comment by mffap** [comment]
> Looks good to me. You links some images that are not in the pr. Not sure if they already exist or if they are missing.

Thanks. Yes it's an image from the console guide with a user profile. Should work.

How can users export events/audit logs into thirdparty applications? Is there a possibility or planned work around it?

**Comment by mffap** [comment]
> How can users export events/audit logs into thirdparty applications? Is there a possibility or planned work around it?

@fforootd we should quickly discuss the requirements and plans on this with @hifabienne 
While it might be possible to do it with Actions, we may want to provide an integrated feature in the mid/longterm

File: docs/docs/concepts/features/audit-trail.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Using logs in external systems
+
+You can use the [Event API](#event-api) to pull data and ingest it in an external system.
+
+[Actions](actions.md) can be used to write events to the stdout and [process the events as logs](../../self-hosting/manage/production#logging).
+Please refer to the zitadel/actions repository for a [code sample](https://github.com/zitadel/actions/blob/main/examples/post_auth_log.js).
+You can use your log processing pipeline to parse and ingest the events in your favorite analytics tool.
+
+Theoretically it would be possible to send events directly with an http request to an external tool.
```
Comment (fforootd) [inline docs/docs/concepts/features/audit-trail.md:50]:
```suggestion
It is possible to send events directly with an http request to an external tool.
```

**Review by fforootd — COMMENTED** [review]
great write up, only some small inputs from my side

File: docs/docs/concepts/features/audit-trail.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+![Event viewer](/img/concepts/audit-trail/event-viewer.png)
+
+### Event API
+
+Since everything that is available in Console can also be called with our APIs, you can access all events and audit data trough our APIs:
+
+- [Event API Guide](/docs/guides/integrate/event-api)
+- [API Documentation](/docs/category/apis/admin/events)
+
+Access to the API is possible with a Service User account, allowing you to integrate the events with your own business logic.
```
[the remark this answers is not in the document]
Comment (mffap) [reply docs/docs/concepts/features/audit-trail.md]: added

**Review by fforootd — APPROVED** [review]

**Pull request merged**